src/Alert/Alert.tsx

```diff
@@ -0,0 +1,106 @@
+/** @jsx jsx */
+import { jsx, css } from "@emotion/core";
+import React, { useMemo } from "react";
+import PropTypes from "prop-types";
+
+import { Card, CardSeperator, CardProps } from "../Card";
+import { IconErrorSolid } from "../icons/IconErrorSolid";
+import { IconInfoSolid } from "../icons/IconInfoSolid";
+import { IconWarningSolid } from "../icons/IconWarningSolid";
```

Comment:
Also the warning icon itself should be using `orange-base` as the triangle bg color.